docs/configs/pdf/index.rst

```diff
@@ -39,6 +39,7 @@ Building applications
    app-dev/bindings-java/index
    app-dev/bindings-scala/index
    app-dev/bindings-js
+   app-dev/bindings-x-lang/index
```

Comment:
I think this should maybe go _after_ the gRPC entry? As gRPC and all of the ones above are things we provide for you, whereas this is striking out on your own.

I wonder if we should also consider combining this new page and the gRPC page into one? Because the user needs are sort of connected together.